shimmingtoolbox/unwrap/prelude.py

```diff
@@ -80,4 +81,9 @@ def prelude(wrapped_phase, mag, affine, mask=None, threshold=None, is_unwrapping
 
     fname_phase_unwrapped = glob.glob(os.path.join(path_tmp, 'rawPhase_unwrapped*'))[0]
 
-    return nib.load(fname_phase_unwrapped).get_fdata()
+    phase_unwrapped = nib.load(fname_phase_unwrapped).get_fdata()
+
+    for _ in range(wrapped_phase.ndim - phase_unwrapped.ndim):
+        phase_unwrapped = np.expand_dims(phase_unwrapped, -1)
```

Comment:
are you sure you need to run this code systematically? ie: the issue we've encountered with the removal of the last dim might be due to the fact that we used single slice scans

Reply:
my comment suggest better unit test coverage (eg run code with single-slice and multi-slice, to make sure expected behaviour is met)

Reply:
> my comment suggest better unit test coverage (eg run code with single-slice and multi-slice, to make sure expected behaviour is met)

All tests in test_prelude introduced before this PR data that have shapes: [128, 76, 10], so they cover the multi-slice scenario. The three tests introduced in this PR have shapes: [128, 76, 1], [128, 1], [128, 1, 1], which cover the single slice scenarios.